packages/passport/sdk/src/starkEx/passportImxProviderFactory.ts

```diff
@@ -33,7 +33,7 @@ export class PassportImxProviderFactory {
 
   private readonly passportEventEmitter: TypedEventEmitter<PassportEventMap>;
 
-  private readonly imxApiClients: ImxApiClients;
+  public readonly imxApiClients: ImxApiClients;
```

Comment:
Correct me if I'm wrong but this class is not exposed to users, therefore no harm in making this public (note still readonly). Allows for testability.